extensions/ims/user.schema.json

```diff
@@ -26,6 +26,23 @@
         "type": {
           "type": "string",
           "const": "https://ns.adobe.com/adobecloudplatform/ims/user"
+        },
```

Comment:
Can you also please fix id->xdm:id and type->xdm:type?  Since this is still an experimental schema, we can change anything...